demisto_sdk/commands/generate_modeling_rules/tests/generate_modeling_rules_test.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+
+from demisto_sdk.commands.generate_modeling_rules.generate_modeling_rules import (
+    array_create,
+    convert_raw_type_to_xdm_type,
+    convert_to_xdm_type,
+    create_xif_header,
+    extract_raw_type_data,
+    json_extract_array,
+    json_extract_scalar,
+    replace_last_char,
+    to_number,
+    to_string,
+)
+
+
+@pytest.mark.parametrize("s, res", (["hello,\n", "hello;\n"], ["", ""]))
+def test_replace_last_char(s, res):
```

Comment:
please add a given/when/then docstring for each test

Reply:
Done